<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>321</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialpad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresources.qrc">
    <normaloff>:/resources/bgd_bubbles.png</normaloff>:/resources/bgd_bubbles.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: url(:/resources/bgd_roster_gradient.png);
background-position: bottom right;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QFrame" name="frameBubbles">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resources/bgd_bubbles.png);
background-position: left;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblCreditText">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: black;
background: none;</string>
          </property>
          <property name="text">
           <string>Credit:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(98, 146, 213);
background: none;</string>
          </property>
          <property name="text">
           <string>$0.00</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(98, 146, 213);
background: none;</string>
          </property>
          <property name="text">
           <string>Buy credit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QFrame" name="frameNumber">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/resources/field_normal.png) 0 0 0 0 stretch stretch;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pbFlag">
             <property name="styleSheet">
              <string notr="true">background: white;
border-image: url(:/resources/bgd_bubbles.png)</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbArrow">
             <property name="styleSheet">
              <string notr="true">margin: 0;
border: 0;
border-image: none;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="myresources.qrc">
               <normaloff>:/resources/combo_arrow.png</normaloff>:/resources/combo_arrow.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditPhoneNumber">
             <property name="font">
              <font>
               <stylestrategy>NoAntialias</stylestrategy>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: none;</string>
             </property>
             <property name="inputMask">
              <string notr="true">+00000000000000</string>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="cursorPosition">
              <number>1</number>
             </property>
             <property name="placeholderText">
              <string>enter phone number</string>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>pbArrow</zorder>
          <zorder>lineEditPhoneNumber</zorder>
          <zorder>pbFlag</zorder>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>500</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/resources/btn_white_hover.png) 0 0 0 0 stretch stretch;</string>
          </property>
          <property name="text">
           <string>   Send  </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditMessage">
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblMessageCharCount">
          <property name="text">
           <string>160</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color: rgb(156, 166, 177);</string>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblNumberOfMessages">
          <property name="styleSheet">
           <string notr="true">color: rgb(156, 166, 177);</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>Call</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color: rgb(98, 146, 213);</string>
          </property>
          <property name="text">
           <string>See rates</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pbFlag</tabstop>
  <tabstop>lineEditPhoneNumber</tabstop>
  <tabstop>textEditMessage</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pbArrow</tabstop>
 </tabstops>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
